brainglobe_atlasapi/atlas_generation/atlas_scripts/perens_stereotaxic_mri_mouse.py

```diff
@@ -0,0 +1,402 @@
+# %%
+
+
+def create_atlas(working_dir, resolution):
+    ATLAS_NAME = "perens_stereotaxic_mri_mouse"
+    SPECIES = "Mus musculus"
+    ATLAS_LINK = "https://www.neuropedia.dk/resource/multimodal-3d-mouse-brain-atlas-framework-with-the-skull-derived-coordinate-system/"
+    CITATION = "Perens et al. 2023, https://doi.org/10.1007/s12021-023-09623-9"
+    ORIENTATION = "ial"
+    ROOT_ID = 997
+    ATLAS_FILE_URL = "https://www.neuropedia.dk/wp-content/uploads/Multimodal_mouse_brain_atlas_files.7z"
+
+    # Temporary folder for  download:
+    download_dir_path = working_dir / "downloads"
+    download_dir_path.mkdir(exist_ok=True)
+    atlas_files_dir = download_dir_path / "atlas_files"
+
+    ## Download atlas_file
+    utils.check_internet_connection()
+
+    import urllib.request
+
+    destination_path = download_dir_path / "atlas_download.7z"
+    headers = {
+        "User-Agent": (
+            "Mozilla/5.0 "
+            "(Windows NT 10.0; "
+            "Win64; x64; "
+            "rv:129.0) "
+            "Gecko/20100101 "
+            "Firefox/129.0"
+        ),
+        "Accept": (
+            "text/html,"
+            "application/xhtml+xml,"
+            "application/xml;q=0.9,"
+            "image/avif,"
+            "image/webp,"
+            "image/png,"
+            "image/svg+xml,"
+            "*/*;q=0.8"
+        ),
+        "Accept-Language": "en-GB,en;q=0.5",
+        "Accept-Encoding": "gzip, deflate, br, zstd",
+        "DNT": "1",
+        "Sec-GPC": "1",
+        "Host": "www.neuropedia.dk",
+        "Connection": "keep-alive",
+        "Upgrade-Insecure-Requests": "1",
+        "Sec-Fetch-Dest": "document",
+        "Sec-Fetch-Mode": "navigate",
+        "Sec-Fetch-Site": "none",
+        "Sec-Fetch-User": "?1",
+        "TE": "trailers",
+        "Priority": "u=0, i",
+    }
+    if not os.path.isdir(
+        atlas_files_dir / "Multimodal_mouse_brain_atlas_files"
+    ):
+        req = urllib.request.Request(ATLAS_FILE_URL, headers=headers)
+        with (
+            urllib.request.urlopen(req) as response,
+            open(destination_path, "wb") as out_file,
+        ):
+            data = response.read()  # a `bytes` object
+            out_file.write(data)
+        with py7zr.SevenZipFile(destination_path, mode="r") as z:
+            z.extractall(path=atlas_files_dir)
+        destination_path.unlink()
+
+    structures_file = (
+        atlas_files_dir
+        / "Multimodal_mouse_brain_atlas_files"
+        / "Hierarchy_tree"
+        / "Annotation_info.csv"
+    )
+    annotations_file = (
+        atlas_files_dir
+        / "Multimodal_mouse_brain_atlas_files"
+        / "MRI_space_oriented"
+        / "mri_ano.nii.gz"
+    )
+    reference_file = (
+        atlas_files_dir
+        / "Multimodal_mouse_brain_atlas_files"
+        / "MRI_space_oriented"
+        / "mri_temp.nii.gz"
+    )
+
+    annotated_volume = sitk.GetArrayFromImage(
```

Comment:
For consistency with other packaging scripts, can we use `brainglobe_utils.IO.image.load_any` instead of ITK here and lines below?

Reply:
done